Document/Testcases/0x00b_OMTG-DATAST_iOS.md

```diff
@@ -1,5 +1,54 @@
-## <a name="OMTG-DATAST-002"></a>OMTG-DATAST-002: Testing for Sensitive Data Disclosure in Log Files
+## <a name="OMTG-DATAST-001-1"></a>OMTG-DATAST-001-1: Test for system credentials storage features 
+
+### Black-box Testing
+
+A way to identify if sensitive information like credentials and keys are stored insecurely and without leveraging the native functions from IOS is to analyse the app data directory. It is important to trigger as much app functionality as possbile before the data is analysed, as the app might only store system credentials as specific functionality is triggered by the user. A static analysis can then be performed for the data dump based on generic keywords and app specifc data. Identify how the application stores data locally on the iOS device. Some of the possible options for the application to store it's data locally includes:
+ 
+- Plain Files
+- Plist
+- SQLite3 DB
+- Realm DB
+
+Steps :
+
+1. Proceed to trigger functionality that stores potential sensitive data.
+2. SSH into your iOS device and browse to the following directory: `/var/mobile/Containers/Data/Application/$APP_ID/`
```

Comment:
SSH is not the only way to connect to a device. You can use file explorer tools and also connect via USB and Needle. It might be good to have a generic section on how to connect to an iOS device and then reference to it whenever you need it.